app/lib/filters/taxon_filter.rb

```diff
@@ -0,0 +1,11 @@
+module Filters
+  class TaxonFilter < Filter
+    def value
+      topic = params['top_level_taxon']
```

Comment:
Could we make this more generic so that it supports any taxon tree?

Knowledge of how the taxon filtering works in the UI, in the form of a primary and secondary dropdown is leaking into the implementation here.

Reply:
We can do though for now I think this suits our purposes, as I don't think we plan to make this facet support any other levels of the taxonomy tree.